tools/cfgsync-cdn/ai.cfgsync.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "Application Insights CfgSync CDN",
+    "description":"Application Insights throttleMgrCfg Configurations and feature opt-in Details",
+    "version": "1.0.0",
+    "homepage": "https://github.com/microsoft/ApplicationInsights-JS#readme",
+    "author": "Microsoft Application Insights Team",
+    "sideEffects": false,
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/microsoft/ApplicationInsights-JS"
+    },
+    "enabled": true,
```

Comment:
add scripts folder, add copy task,

Reply:
and add version number after the json name

Reply:
actually, I created the config folder a long time ago with build task defined and release scripts. so moved the CDN to that folder and changed the version from 0.1.0 to 1.0.0